.github/workflows/test.yml

```diff
@@ -10,7 +10,20 @@ concurrency:
 
 jobs:
   tests:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        target:
+          - id: "linux-aarch64"
+            os: "ubuntu-latest"
+          - id: "linux-amd64"
+            os: "ubuntu-20.04"
+          - id: "darwin-aarch64"
+            os: "macos-latest"
+          - id: "darwin-amd64"
+            os: "macos-13"
```

Comment:
Same as with ubuntu-latest, even if a little more believable because.. Apple

Reply:
This one is valid, macos 13 runs amd64 and macos latest runs aarch64